phac-nml/irida-next
# Fix edit project form errors

## What does this PR do and why?
Updated the `Project name and description` form to display errors. Sidebar & breadcrumbs are out of scope.
Fixes #172.

## Screenshots or screen recordings
![Screenshot from 2023-09-15 13-02-08](https://github.com/phac-nml/irida-next/assets/325703/a9d764b8-57a5-4115-9a73-7460d48d64ed)
![Screenshot from 2023-09-15 13-01-43](https://github.com/phac-nml/irida-next/assets/325703/3d465018-d52a-4dcb-b70d-a8846e694f35)

## How to set up and validate locally
1. Navigate to a Project
2. Click the `Settings` link in the sidebar
3. In the `Project name and description` form set `Project Name` to `a`
4. Click `Update project`
5. See that the page re-renders with `a` now showing in the sidebar instead of the original project name
6. Reload the page and see that the original project name now displays instead of `a`

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
